applications/endoscopy_out_of_body_detection/cpp/metadata.json

```
@@ -10,12 +10,13 @@
 		"language": "C++",
 		"version": "1.0",
 		"changelog": {
-			"1.0": "Initial Release"
+			"1.0": "Initial Release",
+			"1.1": "Update directory structure and CMakeLists"
 		},
 		"holoscan_sdk": {
 			"minimum_required_version": "0.5.0",
```

Suggest dropping 0.5.0 support if no longer supported without nontrivial source changes:

```suggestion
			"minimum_required_version": "1.0.3",
```